rules-tests/TypeDeclaration/Rector/Property/TypedPropertyFromStrictConstructorRector/Fixture/readonly_public_property.php.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Property\TypedPropertyFromStrictConstructorRector\Fixture;
+
+class ReadonlyPublicProperty
```

Comment:
this test-case is not super realistic, but since the readonly flag is only enforced via static analysis its a case which can happen